pyspark/bigdl/nn/layer.py

```diff
@@ -964,6 +964,20 @@ class ExpandSize(Layer):
     def __init__(self, sizes, bigdl_type="float"):
         super(ExpandSize, self).__init__(None, bigdl_type, sizes)
 
+class TransformerLayer(Layer):
+
+    '''
+    Implementation TransformerLayer
```

Comment:
why we should add `Layer` to `Transformer`, it's actually under nn right ?

Reply:
and `Implementation for` or just put `Implementation` after the layer name

Reply:
ok, change to "Implementation for".
As our preprocessing is called "Transformer", so here "TransformerLayer" to separate from preprocessing.

Reply:
it should be fine as it's under different package. From `nn` package's perspective, no need to add `Layer` explicitly

Reply:
ok, changed